kirkwood: add 6.12 as testing kernel

CHKDSK88 opened the pull request:
Support for 6.12 is ready; add it as a testing kernel.

Run tested on Cloud Engines PogoPlug Series 4.
    
Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>

CHKDSK88 committed:
kirkwood: move files-6.6 to files

No need to keep it in kernel specified folder anymore.

Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19069
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

CHKDSK88 committed:
kernel/kirkwood: Create kernel files for v6.12 (from v6.6)

This is an automatically generated commit.

When doing `git bisect`, consider `git bisect --skip`.

Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19069
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

CHKDSK88 committed:
kernel/kirkwood: Restore kernel files for v6.6

This is an automatically generated commit which aids following Kernel patch
history, as git will see the move and copy as a rename thus defeating the
purpose.

For the original discussion see:
https://lists.openwrt.org/pipermail/openwrt-devel/2023-October/041673.html

Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19069
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

CHKDSK88 committed:
kirkwood: kernel: 6.12: refresh patches

Most of them were manually refreshed.

Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19069
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

CHKDSK88 committed:
kirkwood: add 6.12 as testing kernel

Support for 6.12 is ready; add it as a testing kernel.

Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19069
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

sanderpqr commented:
Run tested on ix4-200d, no issues.

hauke commented:
It is currently not building:
```
Framer Subsystem (FRAMER) [N/m/y/?] (NEW) make[7]: 
```

CHKDSK88 committed:
kirkwood: 6.12: refresh kernel config

6.12 kernel config need to be refreshed.

Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19069
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

CHKDSK88 commented:
Done. Apologies for the delay. I was away.
